debian/control

```diff
@@ -0,0 +1,26 @@
+Source: nccl
+Section: universe/science
+Maintainer: Boris Fomitchev <bfomitchev@nvidia.com>
+Priority: optional
+Build-depends: debhelper(>=9), nvidia-352-dev, cuda-toolkit-7-0,
+               cuda-curand-dev-7-0, cuda-cublas-dev-7-0, cuda-cudart-dev-7-0, cuda-ld-conf-7-0
+Standards-Version: 3.9.5
+
+Package: nccl
+Section: universe/tools
+Architecture: amd64
+Depends: ${misc:Depends}, ${shlibs:Depends},  nvidia-352,
```

Comment:
Can confirm that I use the RUN file and know plenty of others do. A range of drivers should ideally be supported.